### DISCUSSION TITLE
Glass Doors & Trapdoor separation

### DISCUSSION THREAD

**Discussion by MicadenTDCT** [BedrockTweaks/Files#94] [Ideas]
Most people use trapdoors for builds that use trapdoors for non-intended purposes, but not for doors. I want to be able to look through doors and not make my trapdoor heavy builds ugly.

**Comment by drav0011** [comment]
This was merged for 2 reasons
First vanilla tweaks did merge both packs so we doing the same
and second compatibility, when making other packs compatible between them having 2 packs which both change wooden items increses the amount of compatibility packs exponentially so it is harder to mantain
